.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
       - image: zeplin/amazon-linux-ami:cci-node12
     steps:
       - checkout
-      - run: sudo npm i -g lerna
+      - run: sudo npm i -g lerna@5.3.0
```

Comment:
I tried npm workspaces, and it looks like it's working as we expected, but before implementing it into our overall workflow, I wanted to experience the release process from scratch since it has some unique steps while publishing.
After finalizing this PR, my plan is;
- To remove `lerna` and use `npm workspaces` 
- To remove `.cirlcleci` and use `github actions`